CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.10.0 (2020-05-06)
+
+- Updated all CLI commands to throw an error, informing the user to uninstall the package globally. [#78](https://github.com/blackbaud/skyux-sdk-builder-plugin-migrate/pull/78)
```

Comment:
"...to display an error that informs users to uninstall..." would be my suggestion. Does "display" work?